server_test.go

```diff
@@ -668,7 +669,7 @@ func init() {
 }
 
 func checkInProgressQueriesAtShutdownServer(t *testing.T, srv *Server, addr string, client *Client) {
-	const requests = 100
+	const requests = 5
```

Comment:
100 is probably overkill in most cases, but 5 might be a bit too few. What about 15 or 25?

Reply:
made it 15 + comment